hrpc/call.go

```diff
@@ -205,6 +205,49 @@ func MaxVersions(versions uint32) func(Call) error {
 	}
 }
 
+// MaxResultsPerColumnFamily sets the maximum number of values returned for each row
```

Comment:
would it be more precise to say: "MaxResultsPerColumnFamily is an option for Get or Scan requests that sets the maximum number of cells returned per each column family in a row"?